mne/beamformer/tests/test_lcmv.py

```diff
@@ -797,10 +798,26 @@ def test_lcmv_maxfiltered():
     epochs = mne.Epochs(raw_sss, events)
     data_cov = mne.compute_covariance(epochs, tmin=0)
     fwd = mne.read_forward_solution(fname_fwd)
+    return epochs, data_cov, fwd
+
+
+@testing.requires_testing_data
+@pytest.mark.parametrize('use_rank', ('info', 'computed', 'full', None))
+def test_lcmv_maxfiltered(mf_data, use_rank):
+    """Test LCMV on maxfiltered data."""
+    epochs, data_cov, fwd = mf_data
     rank = compute_rank(data_cov, info=epochs.info)
     assert rank == {'mag': 71}
-    for use_rank in ('info', rank, 'full', None):
-        make_lcmv(epochs.info, fwd, data_cov, rank=use_rank)
+    ctx = nullcontext()
+    if use_rank == 'computed':
+        use_rank = rank
+    elif use_rank is None:
+        ctx = pytest.warns(RuntimeWarning, match='rank as it exceeds')
+    with catch_logging() as log, ctx:
+        make_lcmv(epochs.info, fwd, data_cov, rank=use_rank, verbose=True)
+    log = log.getvalue()
+    n = 102 if use_rank == 'full' else 71
+    assert f"Making LCMV beamformer with rank {{'mag': {n}}}" in log
```

Comment:
@britta-wstnr yes that old issue was meant to check/fix the beamformer behavior, but it didn't in the case of `rank='info'` (at least for `make_lcmv`) unfortunately. That's probably because we focused mostly on making `estimate_rank` work better, and if you pass `rank=dict(...)` where the `dict` comes from `estimate_rank` things will generally work okay. So the `rank='info'` case must have slipped through the cracks, as this test passes on this PR but fails on `main` for the `rank='info'` case.